rllib/algorithms/ppo/tests/test_ppo_with_rl_module.py

```diff
@@ -176,6 +251,134 @@ def test_ppo_exploration_setup(self):
             check(np.mean(actions), 1.5, atol=0.2)
             trainer.stop()
 
+    def test_rollouts(self):
```

Comment:
For RLModule specific unittests can we create a separate file test_ppo_rl_module.py and place these unittests there? 
So in other words we will have two tests for each algorithm
`test_xxx_algo_with_rl_module.py` and `test_xxx_rl_module.py`

Reply:
Good idea. Done!